### PR TITLE
Add v1.17 Release Blog

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

File: content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+The Kubernetes Storage SIG identified snapshot operations as critical functionality for many stateful workloads. For example, a database administrator may want to snapshot a database volume before starting a database operation.
+
+By providing a standard way to trigger snapshot operations in the Kubernetes API, Kubernetes users can now handle use cases like this without having to go around the Kubernetes API (and manually executing storage system specific operations).
+
+Instead, Kubernetes users are now empowered to incorporate snapshot operations in a cluster agnostic way into their tooling and policy with the comfort of knowing that it will work against arbitrary Kubernetes clusters regardless of the underlying storage.
+
+Additionally these Kubernetes snapshot primitives act as basic building blocks that unlock the ability to develop advanced, enterprise grade, storage administration features for Kubernetes: including application or cluster level backup solutions.
+
+Read the [full release blog here](https://kubernetes.io/blog/2019/12/09/csi-volume-snapshot-beta/).
```
Comment (sftim) [inline content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md]: optional reword:
```suggestion
You can read more in the blog entry about [releasing CSI volume snapshots to beta](https://kubernetes.io/blog/2019/12/09/csi-volume-snapshot-beta/).
```

File: content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+The CSI migration effort enables the replacement of existing in-tree storage plugins such as `kubernetes.io/gce-pd` or `kubernetes.io/aws-ebs` with a corresponding CSI driver. If CSI Migration is working properly, Kubernetes end users shouldn’t notice a difference. After migration, Kubernetes users may continue to rely on all the functionality of in-tree storage plugins using the existing interface.
+
+When a Kubernetes cluster administrator updates a cluster to enable CSI migration, existing stateful deployments and workloads continue to function as they always have; however, behind the scenes Kubernetes hands control of all storage management operations (previously targeting in-tree drivers) to CSI drivers.
+
+The Kubernetes team has worked hard to ensure the stability of storage APIs and for the promise of a smooth upgrade experience. This involves meticulous accounting of all existing features and behaviors to ensure backwards compatibility and API stability. You can think of it like changing the wheels on a racecar while it’s speeding down the straightaway.
+
+Read the [full release blog here](https://kubernetes.io/blog/2019/12/09/csi-migration-beta/).
```
Comment (sftim) [inline content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md]: optional reword:
```suggestion
You can read more in the blog entry about [CSI migration going to beta](https://kubernetes.io/blog/2019/12/09/csi-migration-beta/).
```

**Comment by sftim** [comment]
/retitle [WIP] Add v1.17 Release Blog
There is a FIXME-type  hyperlink: https://github.com/kubernetes/website/pull/18030/commits/b8e7f95937cac4ba2340f47d6e4ea9d87c1b00ce#diff-100ffc3d763b0022c5dbf5f21f59737dR119

**Comment by sftim** [comment]
@rawkode feel free to retitle when that's fixed

File: content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+## Release Team
+
+This release is made possible through the efforts of hundreds of individuals who contributed both technical and non-technical content. Special thanks to the [release team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.17/release_team.md) led by Guinevere Saenger. The 35 individuals on the release team coordinated many aspects of the release, from documentation to testing, validation, and feature completeness.
+
+As the Kubernetes community has grown, our release process represents an amazing demonstration of collaboration in open source software development. Kubernetes continues to gain new users at a rapid pace. This growth creates a positive feedback cycle where more contributors commit code creating a more vibrant ecosystem. Kubernetes has had over [39,000 individual contributors](https://k8s.devstats.cncf.io/d/24/overall-project-statistics?orgId=1) to date and an active community of more than 66,000 people.
+
+
+## Webinar
+
+Join members of the Kubernetes 1.17 release team on Jan 7th, 2020 to learn about the major features in this release. Register [here](??).
```
Comment (sftim) [inline content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md]: This link is missing.

**Comment by rawkode** [comment]
I think this is good.

Pending conversations with CNCF: @onlydole @guineveresaenger @justaugustus

**Comment by castrojo** [comment]
/approve

Remove the hold when you want to go live, nice work!

**Comment by guineveresaenger** [comment]
Yaaaay! Thanks everyone!

File: content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+date: 2019-12-10
+slug: kubernetes-1-17-release-announcement
+---
+
+**Authors:** [Kubernetes 1.17 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.17/release_team.md)
+
+We’re pleased to announce the delivery of Kubernetes 1.17, our fourth and final release of 2019! Kubernetes v1.17 consists of 22 enhancements: 14 enhancements have graduated to stable, 4 enhancements are moving to beta, and 4 enhancements are entering alpha.
+
+# Major Themes
+
```
Comment (kbhawkey) [inline content/en/blog/_posts/2019-12-09-kubernetes-1.17-release-announcement.md]: Hello @rawkode .
nit: formatting: Typically, I _think_ the H1 level header is not used.

**Pull request merged**